packages/bot-web-ui/src/stores/load-modal-store.ts

```diff
@@ -526,6 +531,7 @@ export default class LoadModalStore implements ILoadModalStore {
                 if (load_options.workspace) {
                     (load_options.workspace as any).RTL = isDbotRTL();
                 }
+                this.imported_strategy_type = upload_type as string;
```

Comment:
```suggestion
                this.imported_strategy_type = upload_type;
```
Not needed if return is as per suggested

Reply:
Done 👍